src/app.js

```diff
@@ -0,0 +1,101 @@
+import axios from 'axios';
+
+//API die ik ga gebruiken
+//https://restcountries.com/v3.1/name/{name}
+
+//Stappenplan
+//Er moet een search functie gemaakt worden
+//-Als eerste moet er een event listener gezet worden op het invoerveld
+//-De input van die listener moet op een variabele gezet worden
+//-De variable moet gebruikt worden met de bovenstaande API om te kunnen zoeken
+//De search functie moet op basis van de common name zijn
+//Uit de search functie moet de volgende info gehaald worden:
+//1. De ligging (Western Europe bij NL)
+//2. De populatie
+//3. De hoofdstad
+//4. Waarmee betaald kan worden (ook aangeven indien er meerdere zijn)
+//5. De talen die gesproken worden
+//Dit moet allemaal in een string gezet worden
+//Deze string moet geinjecteerd worden in de DIV
+
+const searchField = document.getElementById('search-field');
+const submitForm = document.getElementById('submit-form');
+const searchData = document.getElementById('search-data')
+const searchError = document.getElementById('form-error-message')
+
+let searchInput = "";
+
+//Kan ook op deze manier geschreven worden:
+
+// searchField.addEventListener('input', (e) => {
+//     return searchInput = e.target.value;
+// });
+
+// submitForm.addEventListener('submit', (e) => {
+//     e.preventDefault();
+//     void fetchCountry(searchInput)
+// })
+
+searchField.addEventListener('input', handleInput)
+submitForm.addEventListener('submit', handleSubmit)
+
+function handleInput(e) {
+    return searchInput = e.target.value
+}
+
+function handleSubmit(e) {
+    e.preventDefault();
+    void fetchCountry(searchInput)
+}
+
+const ENDPOINT = "https://restcountries.com/v2/name/";
+
+async function fetchCountry(searchInput) {
+    try {
+        const response = await axios.get(`${ENDPOINT}${searchInput}`)
+        const countryData = response.data;
```

Comment:
```suggestion
        const { currencies, languages, flags, name, subregion, population, capital } = response.data[0];
```

Dan hoef je niet overal `countryData[0].` te herhalen 😉